alica_engine/include/engine/IAlicaLogger.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <iostream>
+#include <sstream>
+#include <string>
+
+namespace alica
+{
+// IAlicaLogger has nothing to do with engine/Logger.h
+
+enum class Verbosity
+{
+    DEBUG,
+    INFO,
+    WARNING,
+    ERROR,
+    FATAL
+};
+
+class IAlicaLogger
+{
+public:
+    // IAlicaLogger() = default;
+    virtual ~IAlicaLogger() = default;
+
+    template <class... Args>
+    void log(Verbosity verbosity, Args&&... args)
```

Comment:
I would also request a method returning ostream that we can use with << like in Ros
I suppose it might need to print out \n when the ostream destructs

Would also be nice to have convenience functions for different verbosity, like logWarn